code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -113,6 +113,12 @@
 		if(100 to 200)
 			. += span_warning("[t_He] [t_is] twitching ever so slightly.")
 
+	if("wings" in dna.species.mutant_bodyparts)
+		var/hacked = ""
+		if(mind?.martial_art && istype(mind.martial_art, /datum/martial_art/ultra_violence))
+			hacked = ionnum() + ionnum() + " "
+		. += "[t_He] [t_has] also have a pair of [span_warning(hacked)][(dna.features["wings"])] wings on their back"
```

Comment:
Blood drenched wings perhaps? Hacked is something strange to say someone just sees